### PR TITLE
beta fix

### PR DESCRIPTION
**Pull request by Kamill90**

### CODE REVIEW

**Comment by marek-siemieniuk-morawski** [comment]
How does it solve problems with Beta version?

**Comment by Kamill90** [comment]
> How does it solve problems with Beta version?

There is a proper beta 'MainActivity.java file that needs to be pasted

**Review by marek-siemieniuk-morawski — APPROVED** [review]

File: app.json [hunk 1]
```diff
@@ -1,4 +1,4 @@
 {
-  "name": "GoldWallet",
-  "displayName": "GoldWallet"
+  "name": "Testnet Gold Wallet",
```
Comment (KedziaPawel) [inline app.json:2]: does this affect in any way the app?

Comment (marek-siemieniuk-morawski) [reply app.json:2]: In my opinion it doesn't

**Review by KedziaPawel — APPROVED** [review]

**Comment by KedziaPawel** [comment]
will be done in new flow

**Pull request closed without merging**